internal/config/testdata/valid_config.yaml

```diff
@@ -31,3 +31,5 @@ enable-hns: true
 file-system:
   ignore-interrupts: true
   disable-parallel-dirops: true
+monitoring:
```

Comment:
i think this should be called as observability than monitoring.

@kislaykishore wdyt ?

Reply:
We are moving stackdriver params under "metrics". So, we should put this too there.

Reply:
Done.